integration/__tests__/command-palette.tests.ts

```diff
@@ -0,0 +1,36 @@
+import { Application } from "spectron";
```

Comment:
Refactored integration tests to separate files so that they are easier to understand & run in a logical units.

Reply:
I would prefer to keep it here. It was almost impossible to refactor/add integration tests when they were in one monolithic test suite (which was also broken at the time of writing).